example/components/PubNubTime.tsx

```diff
@@ -1,17 +1,22 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { usePubNub } from '../../src/index';
 
 const PubNubTime = () => {
-  const PubNubClient = usePubNub();
+  const client = usePubNub();
```

Comment:
I'd suggest using the variable name "pubnub" instead of "client."  We use this in a bunch of other examples.